### PR TITLE
Final desktop

### PR DESCRIPTION
**Pull request by znation2010**
In this milestone I:

- Utilized Media Queries
- Used Grid
- used flexbox

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,401 @@
...
+        <div class="cards-description desc-right">
+          <h2 class="card-titles">Tonic</h2>
+          <div class="subheading-lists">
+            <span class="canopy">CANOPY</span>
+            <img src="assets/icons/circle-icon.svg" alt="circlular icon" />
+            <span class="subheading-list-title">Back End Dev</span>
+            <img src="assets/icons/circle-icon.svg" alt="circlular icon" />
+            <span class="subheading-list-title">2015</span>
+          </div>
+          <section class="card-content">
```
Comment (koachcode) [inline index.html]: - @znation2010 Please for all your project cards, I noticed on the desktop version the spacing between the project name, stack, and bullet point is small compared to that of the Figma design as they are squashed up together in your design, Please refer to the Figma design and add the appropriate spacing a=so as to give your UI a goos loom and to follow details of the Figma design. 

- [Optional] Also I noticed the color of the text and bullet point is `#7a869a` on your design, but on the Figma design it is `#344563`, Please use the appropriate font-size, color, and font-weight as much as possible with reference from the Figma design. 

##### Your design

image

##### Figma design 

image

File: index.html [hunk 2]
```diff
@@ -0,0 +1,401 @@
...
+          <button class="see-project" type="button">See project</button>
+        </div>
+      </div>
+      <div class="cards">
+        <div class="right-card-img">
+          <img
+            class="snapshoots"
+            src="assets/snapshoots/Snapshoot Portfolio-4.svg"
+            alt="Snapshoot svg"
+          />
```
Comment (koachcode) [inline index.html:136]: - I noticed the images of the project do not take the full height as that of the Figma design, please make the image have a full height as that if the Figma design. You can apply the styles below to your `snapshot class` to achieve this.

```css
    height: 100%;
    object-fit: cover;
```

##### Your design 

image


##### Figma Design

image

File: index.html [hunk 3]
```diff
@@ -0,0 +1,401 @@
...
+        </div>
+        <div class="cards-description desc-left">
+          <h2 class="card-titles">
+            Multi-Post <br />
+            Stories
```
Comment (koachcode) [inline index.html:141]: - This project step is about the desktop version and in doing so the mobile version you built previously should not be broken, I noticed on the mobile screen the cards have a margin-left and push it all the way to the right, unlike what we have on figma with the card positioned center of the screen width. 
**Hint**
- You can achieve the center card by making the card have the CSS property below( adjust the width appropriately), first remove the `margin-left` and `margin-right` you had there initially, but leave the margin-bottom. Add the below snippet before the margin-bottom so as to not override it

```css
width: 90%;
margin: 0 auto;
```
##### Your design 

image

##### Figma Design

image

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,491 @@
...
+}
+
+.social-media-links a {
+  margin-right: 1.188rem;
+}
+
+ul {
+  list-style-type: none;
+  display: flex;
+}
```
Comment (koachcode) [inline styles.css]: - I noticed that the stack from the project looks squashed in on the mobile screen and breaks the text, You can use a `flex wrap` property on the `ul` to make it look on on the mobile screen. Check out this [article](https://developer.mozilla.org/en-US/docs/Web/CSS/flex-wrap) to understand what to do, in other to fix it. 

##### Your design 
image

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,491 @@
...
+    font-family: Poppins, serif, sans serif;
+    padding-left: 32px;
+    color: #344563;
+  }
+
+  .cards {
+    display: grid;
+    gap: 20px;
+    grid-template-columns: 1fr 1fr;
+  }
```
Comment (koachcode) [inline styles.css:459]: - On the desktop view, you are supposed to add a hover effect on the card, as indicated on the Figma design, please add the hover effect properties as it is on the Figma.

##### Figma Deign to show hove effect (The blue border color around it)
```css
border: 2px solid #A7AEFF;
```

image

**Review by koachcode — CHANGES_REQUESTED** [review]
Hi @znation2010 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
you-are-near

## Highlights
- No linter Errors :+1: 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by usorfaitheloho — APPROVED** [review]
Hi @znation2010 👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**